css/media.css

```diff
@@ -0,0 +1,3 @@
+@media (max-widht: 767px) {
+  
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)